spring-boot-test-autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -45,6 +45,15 @@ org.springframework.boot.autoconfigure.jdbc.JdbcTemplateAutoConfiguration,\
 org.springframework.boot.autoconfigure.liquibase.LiquibaseAutoConfiguration,\
 org.springframework.boot.autoconfigure.transaction.TransactionAutoConfiguration
 
+# AutoConfigureJooqc auto-configuration imports
```

Comment:
I don't understand that remark. The duplication is already there in several places if I'm not wrong. The proposed `JooqTest` is essentially the same as `JdbcTest`, true. If you point me to the correct way to import everything that `JdbcTest` configures and add the `AutoConfigureJooq` I'd be more to happy to implement it.